docs/intro.rst

```diff
@@ -239,14 +261,38 @@ environment. You can detect that bit size as follows:
   Swig at their installation time. Therefore, there are no prerequisite
   OS-level packages to install.
 
-* Install pywbem (and its prerequisite Python packages) into the active
+* Install pywbem (and its other prerequisite Python packages) into the active
   Python environment:
 
   .. code-block:: bash
 
       > pip install pywbem
 
 
+.. _`Installing to a UNIX-like environment under Windows`:
+
+Installing to a UNIX-like environment under Windows
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This section describes the installation of pywbem into a UNIX-like environment
+under Windows (such as CygWin, MinGW, Babun, or Gow).
+
+Note that Unix-like environments on Windows bring their own Python, so in such
+an environment, you install into that Python, and not into the Python of
+Windows.
+
+* Install pywbem (and its other prerequisite Python packages) into the active
+  Python environment:
+
+  .. code-block:: bash
+
+      > pip install pywbem
```

Comment:
Add something to reference how pywbem is installed from github.  It is discussed in following section.

Reply:
That would apply to all target OSs. Plus, the section about using a virtual python env would also apply to all of them. Atthe end, there are three dimensions (target OS, python env, pywbem version) and the cross product of them can actually be used. I don't think we should start listing that cross product. So far, the description starts simple (with just the list of target OSs), and then the reader learns that there are two more dimensions. Each new dimension is introduced by explaining that it applies to all of the variations before.

If we stick with this approach, I don't think we should add the github information to the section about the UNIX-like env.